cbtdoc.go

```diff
@@ -273,7 +273,7 @@ Usage:
 
 	    Example: cbt createinstance my-instance "My instance" my-instance-c1 us-central1-b 3 SSD
 
-# Create a backup from a source table (deprecated)
```

Comment:
why is this being changed in the docs? Seems to only effect command descriptions that contain `(deprecated)`